src/cookie-io.js

```diff
@@ -100,7 +100,11 @@ class CharacterData {
     }
 
     addClanMember(obj) {
-        this.characterData.clan.members.push(obj);
+        if (obj in this.characterData.clan.members) {
```

Comment:
Didn't know you could use `in` like in python, nice. Should rename `obj` -> `key` or `memberName` to be more accurate though

Reply:
https://github.com/EthanVieira/rs-clicker/pull/71/commits/8e37d85adae8b81c4d9a1d9b3e40c7f746834278